pkg/controller/migmigration/restore.go

```diff
@@ -222,8 +224,25 @@ func (t *Task) buildRestore(backupName string) (*velero.Restore, error) {
 // Update a Restore as desired for the destination cluster.
 func (t *Task) updateRestore(restore *velero.Restore, backupName string) {
 	restorePVs := true
+
+	namespaceMapping := make(map[string]string)
+	for _, namespace := range t.namespaces() {
+		mapping := strings.Split(namespace, ":")
+		if len(mapping) == 1 {
+			namespaceMapping[namespace] = namespace
+		}
+		if len(mapping) == 2 {
+			if mapping[1] == "" {
+				namespaceMapping[mapping[0]] = mapping[0]
```

Comment:
I get why this check is here, but it is weird that a user could validly enter `test:` and have it be valid.